src/active_handout_plugins/exercise.py

```diff
@@ -0,0 +1,12 @@
+from .question import QuestionAdmonition
+
+
+class ExerciseAdmonition(QuestionAdmonition):
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__('exercise', [], *args, **kwargs)
+
+    def create_question_form(self, el, submission_form):
+        return '''
```

Comment:
It would be nice to have this as a template file (even better if it could be overwritten by the user :)).